mintlify/mint
# Get single tenancy to deploy

## Summary
I tried getting single tenancy to deploy for preview deployments and now also for password protected docs. Although the build was outputting that it was building correctly everything was returning a 404. I realized this was happening because the middleware was gating single tenancy from deploying.

### To-Do
- [ ] Fix redirects. I have been looking into this for far too long. I'm just going to move on for now. It works with `yarn build && yarn start` locally. idk why the redirects aren't working when deployed. For example: https://preview.mintlify.com/ isn't deploying. Anyone know how to debug?

## Test Plan
- Make sure code makes sense
- Check out https://preview.mintlify.com/quickstart
